book/whats_new/whats_new.md

```diff
@@ -0,0 +1,4 @@
+# What's New
+
+## August 13 - 2019
+* [Swift 5.0 Migration document](/whats_new/aug-18/swift-5.0-migration.md)
```

Comment:
I would change it something like: "Migrating your plugin to Swift 5.0 and Zapp-iOS v12"

Reply:
done